### ISSUE TITLE
Adding `null` to Gauge.writeMessage crashes the runner

### ISSUE THREAD

**Issue by BugDiver** [getgauge/gauge-java#194]
### Expected behavior
The runner should not be crashing 

### Actual behavior
The runner crashes when a null value is added to 

### Steps to reproduce
1. `gauge init java`
2. Update step implementation to have `Gauge.writeMessage(null);` 
3. `gauge run specs`

It throws following error and crashes the runner 
```
java.lang.NullPointerException: Element at index 0 is null.
    at com.google.protobuf.AbstractMessageLite$Builder.addAllCheckingNulls(AbstractMessageLite.java:367)
    at com.google.protobuf.AbstractMessageLite$Builder.addAll(AbstractMessageLite.java:425)
    at gauge.messages.Spec$ProtoExecutionResult$Builder.addAllMessage(Spec.java:25758)
    at com.thoughtworks.gauge.MessageCollector.addPendingMessages(MessageCollector.java:33)
    at com.thoughtworks.gauge.MessageCollector.addPendingMessagesTo(MessageCollector.java:28)
    at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:49)
    at com.thoughtworks.gauge.connection.MessageDispatcher.dispatchMessages(MessageDispatcher.java:92)
    at com.thoughtworks.gauge.GaugeRuntime.dispatchMessages(GaugeRuntime.java:98)
    at com.thoughtworks.gauge.GaugeRuntime.lambda$connectSynchronously$1(GaugeRuntime.java:81)
    at java.base/java.lang.Thread.run(Thread.java:844)
```
### Gauge version
```
Gauge version: 1.0.4
Commit Hash: 5e997d2

Plugins
-------
csharp (0.10.6)
html-report (4.0.7)
java (0.7.0)
js (2.3.4)
python (0.3.5)
screenshot (0.0.1)
xml-report (0.2.0).
```

**Comment by BugDiver** [comment]
Blocks https://github.com/getgauge/gauge/issues/1248

**Comment by sguptatw** [comment]
[Blocker] - The latest nightly available is 0.7.0.nightly-2018-11-28
